infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuReplicationDiscoveryService.java

```diff
@@ -222,6 +222,10 @@ public void processEvent(DiscoveryServiceEvent event) {
                 processUpgrade(event);
                 break;
 
+            case ENFORCE_SNAPSHOT_FULL_SYNC:
```

Comment:
Is this event only handled by the leader node? Or it is a broadcast?

Reply:
Only leader node.